src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java

```diff
@@ -969,6 +969,25 @@ public void testExportFilters() throws Exception {
 
     }
 
+    @Test
+    public void testComputeFiltersComplexity() throws Exception {
+        MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
+        params.add("networkUuid", NETWORK_UUID.toString());
+        params.add("variantId", VARIANT_ID_1);
+        final var json = "{\"monitoredBranches\":[\"c88c9510-15b4-468d-89c3-c1c6277966c3\"],\"injections\":[\"1110b06b-9b81-4d31-ac21-450628cd34ff\"],\"contingencies\":[\"f631034b-ba7c-4bb8-9d61-258a871e9265\"]}]";
+
+        Map<String, List<Integer>> filtersComplexityCount = objectMapper.readValue(
+                mvc.perform(post("/" + FilterApi.API_VERSION + "/filters/count")
+                                .params(params)
+                                .content(json)
+                                .contentType(APPLICATION_JSON))
+                        .andExpect(status().isOk())
+                        .andReturn().getResponse().getContentAsString(),
+                new TypeReference<>() {
+                });
+        assertEquals(3, filtersComplexityCount.size());
```

Comment:
I don't understand the test : 
Where are built the filters with the uuids c88c9510-15b4-468d-89c3-c1c6277966c3, 1110b06b-9b81-4d31-ac21-450628cd34ff, like in the other unit test methods ...

And contingencies are not filters ...

Reply:
done